pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -157,6 +157,7 @@ func (r *ScaleREST) Update(ctx genericapirequest.Context, name string, objInfo r
 		return nil, false, err
 	}
 
+	// TODO: should we pass admission here?
```

Comment:
I think you've wired this correctly by passing admission a few lines down on the update to the registry.

Reply:
right, will drop the comment